docs/docsite/rst/dev_guide/developing_modules_best_practices.rst

```diff
@@ -190,4 +235,4 @@ Avoid creating a module that does the work of other modules; this leads to code
 Avoid creating 'caches'. Ansible is designed without a central server or authority, so you cannot guarantee it will not run with different permissions, options or locations. If you need a central authority, have it on top of Ansible (for example, using bastion/cm/ci server or tower); do not try to build it into modules.
 
 Always use the hacking/test-module script when developing modules and it will warn
-you about these kind of things.
\ No newline at end of file
+you about these kind of things.
```

Comment:
Suggest revising this sentence to something like "Use the hacking/test-module script when developing modules to receive warnings about these issues".